assets/css/style.css

```diff
@@ -0,0 +1,192 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+}
+
+/* body css */
+body {
+  margin: 0%;
+  font-size: 10px;
+  background-image: linear-gradient(black, grey, silver);
+  min-height: 100vh;
+  font-family: Arial, Helvetica, sans-serif;
+  line-height: 1.2rem;
+  color: white;
+}
+
+/* header css */
+header {
+  position: static;
+  top: 0%;
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  margin-top: 5.6rem;
+}
+
+/* nav container */
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.nav-link {
+  margin: 1.8rem;
+}
+.nav-link a {
+  text-decoration: none;
+  font-size: 1.8rem;
+  color: darkgray;
+}
+
+/* About-Me */
+.about-me-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  align-content: space-around;
+}
+
+.about-me-name,
+.statement-header {
+  margin: 2rem;
+  font-size: 3rem;
+  color: grey;
+  font-style: arial;
+  padding: 5px;
+  text-align: center;
+}
+
+.statement-header {
+  text-decoration: underline;
+  margin-top: 5px;
+  font-size: 2rem;
+}
+
+.statement-info {
+  text-align: center;
+  font-size: 2rem;
+  color: whitesmoke;
+  letter-spacing: 2px;
+  text-indent: 45px;
+  line-height: 30px;
+  margin: 2rem;
+}
+
+.skills {
+  text-decoration: underline;
+  font-size: 2rem;
+  margin: 0%;
+}
+.skill-set,
+.skills {
+  text-align: center;
+  list-style-type: none;
+  font-size: 2rem;
+  letter-spacing: 4px;
+  line-height: 32px;
+  color: whitesmoke;
+}
+/* Project Cards */
+
+.projects-header,
+.contact-header {
+  text-align: center;
+  font-size: 4rem;
+}
+
+.project-title {
+  text-align: center;
+  font-style: arial;
+  font-size: 3rem;
+  padding: 5px;
+  color: rgb(0, 0, 0);
+  text-indent: 45px;
+  line-height: 30px;
+  margin: 4rem;
+  text-decoration: underline;
+}
+
+.project-info {
+  text-align: center;
+  letter-spacing: 2px;
+  text-indent: 45px;
+  line-height: 30px;
+  margin: 2rem;
+  font-size: 2rem;
+}
+
+.project-image {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  object-fit: contain;
+  width: 100%;
+}
+.images {
+  width: 100%;
+  height: 200px;
+}
+.button {
+  border: 3px solid black;
+  width: 30px;
+  padding: 5px;
+  font-size: 17px;
+}
+.button:hover {
+  background-color: antiquewhite;
+}
+
+.first-project-card {
+  border-style: solid;
+  border-color: black;
+  border-width: 9px;
```

Comment:
9px is a very thick border. Consider keeping these a bit sharper, maybe 5px or below.